app/helpers/tags_helper.rb

```diff
@@ -1,2 +1,15 @@
 module TagsHelper
+  def tag_description(tag)
+    if tag.posts.present?
+      tag.posts.count > 1 ? "A collection of #{tag.posts.count} posts" : "A collection of one post"
```

Comment:
I would change this so that the description is not stateful - twitter and facebook will cache the preview meaning if someone shares it with zero posts and then adds posts, it wont update until its manually triggered.

Reply:
updated! 

new design (also updated in the PR's first comment)

<img width="500" alt="Design System 2020-12-21 21-05-13" src="https://user-images.githubusercontent.com/1177031/102859731-8e9f5f80-43d0-11eb-8cf4-21365e895236.png">